AUTHORS

```diff
@@ -0,0 +1,10 @@
+
+Moral rights:
```

Comment:
In questo caso non serve prependere "Moral Rights": state lavorando a titolo personale dunque siete detentori sia dei diritti di proprietà che del diritto morale. La dicitura Moral Rights compare se ad esempio io scrivo codice per conto di un'azienda, che deterrà i diritti finali sulla proprietà del codice, ma rimane il riconoscimento dei diritti morali, inalienabili.

Reply:
Ottimo, grazie del chiarimento!

Reply:
Ultima notazione, di carattere solamente stilistico. Di solito ci piace adottare la convenzione "hacker" per i nomi, e ove desiderato per l'aggiunta di mail, ovvero
`Nome Cognome`
oppure
`Nome Cognome <nome.cognome@email.com>`

La motivazione è la semplice uniformità di stile con gli altri progetti di Developers Italia e quasi tutti i progetti open. Se questo cambiamento era stato effettuato con una possibile motivazione antispam, considera che in ogni caso [at] viene riconosciuto dai crawler, e questi ultimi girano sempre meno frequentemente a caccia di indirizzi (in quanto pratica tendenzialmente inefficace). Essendo una pratica stilistica, suggerisco che venga adottata, ma questo suggerimento rientra in una categoria differente rispetto a quello precedente.

Reply:
@ruphy mi sembra sensato :)
Più che una scelta vera e propria si è trattato di mettere un segnaposto per indicare il fatto che i dati erano stati inseriti di fretta per aprire la PR per partecipare alla wise turtle. Sono d'accordo sull'adottare il formato classico :+1: .

ps: sì, ormai è difficile che i crawler vadano a caccia di email :D